.github/workflows/serverless-pre-release-qa.yml

```diff
@@ -0,0 +1,93 @@
+      E2E_DEV_ACCOUNT_SID: ${{ secrets.E2E_DEV_ACCOUNT_SID }}
+      E2E_DEV_AUTH_TOKEN: ${{ secrets.E2E_DEV_AUTH_TOKEN }}
+    with: 
+      send-slack-message: 'false'
+  run-e2e-tests:
+    needs: build-and-deploy
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      # Clone flex-plugins repo
+      - name: Clone flex-plugins repo
+        run: git clone https://github.com/techmatters/flex-plugins
+        shell: bash
+      # Build Playwright
+      - name: Install e2e-tests dependencies
+        run: npm install
+        working-directory: ./flex-plugins/e2e-tests
+      - name: Setup dependencies for playwright/browsers
+        uses: microsoft/playwright-github-action@v1
+      - name: Install Playwright CLI
+        run: npx playwright install
+        working-directory: ./flex-plugins/e2e-tests
+      # Run E2E tests against actual E2E Flex instance
+      - name: Run Playwright tests
+        run: DEBUG=pw:api PLAYWRIGHT_BASEURL=${{secrets.PLAYWRIGHT_BASEURL_E2E}} PLAYWRIGHT_USER_USERNAME=${{secrets.PLAYWRIGHT_USER_USERNAME}} PLAYWRIGHT_USER_PASSWORD=${{secrets.PLAYWRIGHT_USER_PASSWORD}} TWILIO_ACCOUNT_SID=${{secrets.E2E_DEV_ACCOUNT_SID}} TWILIO_AUTH_TOKEN=${{secrets.E2E_DEV_AUTH_TOKEN}} npx playwright test
+        working-directory: ./flex-plugins/e2e-tests
+      # Upload artifacts
+      # TODO: this is not working and cant tell why :(
+      - uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: test-artifacts
+          path: flex-plugins/e2e-tests/test-results
+
+  generate-pre-release:
+    needs: run-e2e-tests
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Create pre release
+        uses: techmatters/flex-plugins/.github/actions/generate-pre-release@gian_CHI-1262-p2
+        with:
+          tag-prefix: ${{ inputs.tag-prefix }}
+          tag-suffix: 'qa'
+          title: ${{ inputs.title }}
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+        id: create_pre_release
+
+      # Send Slack notifying success
+      - name: Slack aselo-deploys channel
+        uses: techmatters/flex-plugins/.github/actions/notify-deploys@gian_CHI-1262-p2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+          slack-message: "`[Flex]` Action ${{ github.workflow }} completed with SHA ${{ github.sha }}. Release tag is ${{ steps.create_pre_release.outputs.generated-pre-release-tag }} :rocket:."
+        env:
+          SLACK_BOT_TOKEN: ${{ env.GITHUB_ACTIONS_SLACK_BOT_TOKEN }}
```

Comment:
@stephenhand Probably the way we want to use this actions for now? Until we move them to a more reasonable registry of course.